riotfile.py

```diff
@@ -619,16 +643,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     pkgs={"django": [">=2.0,<2.1"]},
                 ),
                 Venv(
-                    pys=select_pys(min_version="3.6"),
-                    pkgs={
-                        "django": [
-                            ">=2.1,<2.2",
-                            ">=2.2,<2.3",
-                        ],
-                    },
+                    pys=select_pys(min_version="3.6", max_version="3.7"),
```

Comment:
Maybe revert these changes? They don't seem relevant to 3.11 (unless the suites actually fail).